src/services/searchArticles.js

```diff
@@ -0,0 +1,16 @@
+import fetch from 'node-fetch';
```

Comment:
If you wanted to, you can combine your fetches in the same file. Totally a preference thing!